pkg/compiler/compiler.go

```diff
@@ -210,53 +210,57 @@ func (comp *compiler) structIsVarlen(name string) bool {
 
 func (comp *compiler) parseIntAttrs(descs map[string]*attrDesc, parent ast.Node,
 	attrs []*ast.Type) map[*attrDesc]uint64 {
-	intAttrs, _ := comp.parseAttrs(descs, parent, attrs)
+	intAttrs, _, _ := comp.parseAttrs(descs, parent, attrs)
 	return intAttrs
 }
 
 func (comp *compiler) parseAttrs(descs map[string]*attrDesc, parent ast.Node, attrs []*ast.Type) (
```

Comment:
Please also add some parsing tests to `pkg/compiler/testdata`.

E.g. some valid descriptions with an attribute to `all.txt` and some tests that should trigger errors to `errors2.txt`.